src/components/App/App.js

```diff
@@ -32,23 +34,33 @@ function App() {
       });
   }, []);
 
-  console.log('movies', movies)
-  console.log('description', topDescription)
+  function fetchSelectedMovie(movieId) {
+    fetchSingleMovie(movieId)
+    .then(data => {
+      const targetMovie = data.movie
+      setSingleMovie(targetMovie)
+    })
+  }
+
+  function handleOpen(movieId) {
+    setSingleMovieId(movieId)
+    fetchSelectedMovie(movieId)
+    setOpen(true)
+  }
 
-  function fetchSelectedMovie(movieId, movieTitle) {
-    console.log(`Movie ID - ${movieId}`)
-    console.log(`Movie Title - ${movieTitle}`)
-    return movieId
+  function handleClose() {
+    setSingleMovieId(0)
+    setOpen(false)
   }
 
   return (
     <main className="App">
       <nav className="Nav-bar"></nav>
       {movies.length > 0 ? (
         <>
-        <Modal />
-        <TopMovie movies={movies} topDescription={topDescription}/>
-        <Movies movies={movies} fetchSelectedMovie={fetchSelectedMovie}/>
+        {singleMovieId !== 0 && <Modal handleClose={handleClose} open={open} movie={singleMovie}/>}
+        {singleMovieId === 0 && <TopMovie movies={movies} topDescription={topDescription} />}
```

Comment:
Could the conditional rendering on 62 && 63 be combined? Both are based on the same boolean logic.